target/linux/brcm63xx/image/bcm63xx.mk

```diff
@@ -549,6 +549,19 @@ define Device/DSL274XB-F1
 endef
 TARGET_DEVICES += DSL274XB-F1
 
+define Device/DSL2750U-C1
+  $(Device/bcm63xx)
+  IMAGES += sysupgrade.bin
+  DEVICE_TITLE := D-Link DSL-2750U rev C1
```

Comment:
Please split up DEVICE_TITLE into
```
DEVICE_VENDOR := D-Link
DEVICE_MODEL := DSL-2750U
DEVICE_VARIANT := C1
```
(not sure about about "rev C1" vs. "C1" here)

See https://github.com/openwrt/openwrt/commit/8c3dbaf8b49233e275b2dce4e30dc0a2bf8c86c5